packages/editor/src/client/routes/articleList.tsx

```diff
@@ -305,17 +306,18 @@ export function ArticleList() {
             )}
 
             <DescriptionListItem label={t('articles.panels.createdAt')}>
-              {currentArticle?.createdAt && new Date(currentArticle.createdAt).toLocaleString()}
+              {currentArticle?.createdAt &&
+                dateTimeWithoutSecondsLocalString(new Date(currentArticle.createdAt))}
             </DescriptionListItem>
 
             <DescriptionListItem label={t('articles.panels.updatedAt')}>
               {currentArticle?.latest.updatedAt &&
-                new Date(currentArticle.latest.updatedAt).toLocaleString()}
+                dateTimeWithSecondsLocalString(new Date(currentArticle.latest.updatedAt))}
             </DescriptionListItem>
 
             {currentArticle?.latest.publishedAt && (
               <DescriptionListItem label={t('articles.panels.publishedAt')}>
-                {new Date(currentArticle.createdAt).toLocaleString()}
+                {dateTimeWithoutSecondsLocalString(new Date(currentArticle.createdAt))}
```

Comment:
I recommend to also use the i18next translation service to format the date. Might be a bit more work if it's just the date but it's cleaner this way.